### ISSUE TITLE
Get a list of EntityHandles for any geometry type

### ISSUE THREAD

**Issue by gonuke** [svalinn/dagmc_stats#1]
Write a function that will return a list of `EntityHandles` that refer to all the `EntitySets` that represent a given geometric type (e.g. volume, surface, curve, node) in a model

**Comment by gonuke** [comment]
Psuedo-code:

```
assume: already loaded model into a MOAB instance "mb"
inputs: none
outputs: list of entityHandles

def get_all_volumes()

     geom_tag = mb.get_tag_handle('GEOM_DIM',....)
     dim = 3
     all_volumes = mb.get_entities_by_tag_and_type(geom_tag, 0, dim)

    return all_volumes
```

**Comment by gonuke** [comment]
This function may not be necessary since it is ultimately a single pyMOAB function call

**Comment by gonuke** [comment]
Closed because it's handled by #3
